pkg/infrastructure/clusterapi/ignition.go

```diff
@@ -173,11 +174,13 @@ func addLoadBalancersToInfra(platform string, config *igntypes.Config, publicLBs
 			// replace the contents with the edited information
 			config.Storage.Files[i].Contents.Source = &encoded
 
+			// Reset error state
+			updateError = nil
+
 			break
```

Comment:
it seems more intuitive to me, that we would return nil here and return error at the end of the function